lerna.json

```diff
@@ -7,7 +7,8 @@
     "samples/21.react-media-template",
     "samples/22.react-agile-poker",
     "samples/23.react-live-canvas",
-    "samples/31.live-canvas-tests"
+    "samples/31.live-canvas-tests",
+    "samples/ts/21.react-media-template"
```

Comment:
After looking at BotFramework samples, I think it's better to expand this to `typescript`. 
BF samples also puts `javascript` -- @Stevenic do we want to follow suit?

https://github.com/microsoft/BotBuilder-Samples/tree/main/samples

Reply:
I can change the name to typescript. Ideally i think the other samples could go in a javascript folder, but did not want to do this as part of this PR